classes/Http/Form/ResetFormType.php

```diff
@@ -19,7 +19,7 @@
 use Symfony\Component\Form\Extension\Core\Type\RepeatedType;
 use Symfony\Component\Form\FormBuilderInterface;
 
-class ResetForm extends AbstractType
+class ResetFormType extends AbstractType
```

Comment:
As the abstract class suggests, this isn't actually a form, but rather a form type, a definition of a form.